src/AutoPilotPlugins/APM/APMSubFrameComponent.qml

```diff
@@ -25,9 +25,20 @@ SetupPage {
     pageComponent:      subFramePageComponent
 
     property bool _oldFW:   globals.activeVehicle.versionCompare(3 ,5 ,2) < 0
+    property var frameModelSelected: undefined
 
     APMAirframeComponentController { id: controller; }
 
+    property Fact _frameConfig: controller.getParameterFact(-1, "FRAME_CONFIG")
+
+    function setFrameConfig(frameNumber) {
+        _frameConfig.value = frameNumber
+    }
+
+    function setFrameParameter(frameNumber) {
```

Comment:
This function name sounds off. What about `loadFrameDefaultParameters`?

Reply:
Done.